test/fixtures/coverage/functional_test/test/pubspec.yaml

```diff
@@ -1,7 +0,0 @@
-name: functional_test_changeme
```

Comment:
This file I actually did remove - previously I remember it being necessary to have a separate pubspec.yaml wherever the functional tests were located because it required an older version of the test package - but that no longer seems to be the case. I removed it and it worked, so I think it's fine now?